app/controllers/adl_document_provider.rb

```diff
@@ -0,0 +1,10 @@
+class AdlDocumentProvider < ::OAI::Provider::Base
+  repository_name 'Adl OAI provider'
+  repository_url  'http://localhost/provider'
+  record_prefix ''
+  admin_email 'root@localhost'   # String or Array
```

Comment:
as above... I think @xenma is adding a configurable admin email for her feedback form - should it be the same?